chain-core/src/state/tendermint.rs

```diff
@@ -224,4 +224,9 @@ impl TendermintVotePower {
     pub fn zero() -> Self {
         TendermintVotePower(0)
     }
+
+    /// get number value of voting power
+    pub fn value(self) -> i64 {
+        self.0
+    }
```

Comment:
this one is redundant -- `i64::from(x)` or `x.into()` should work

Reply:
Fixed